velox/row/tests/UnsafeRowBatchDeserializerTest.cpp

```diff
@@ -668,26 +667,33 @@ class UnsafeRowComplexBatchDeserializerTests
         {intVector, stringVector, intArrayVector, stringArrayVector}, isNullAt);
   }
 
+  void testVectorSerde(const RowVectorPtr& inputVector) {
```

Comment:
Can we consolidate all tests in velox/serializers/tests/UnsafeRowSerializerTest.cpp ?